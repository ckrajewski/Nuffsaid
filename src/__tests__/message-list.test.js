import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import MessageList from '../components/message-list';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('MessageList component', () => {
  test('renders', () => {
    const tree = renderer.create(<MessageList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});