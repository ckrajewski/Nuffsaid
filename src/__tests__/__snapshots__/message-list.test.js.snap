// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageList component renders 1`] = `
<div>
  <div
    className="makeStyles-container-7"
    style={
      Object {
        "visibility": "hidden",
      }
    }
  >
    <div
      className="MuiPaper-root makeStyles-paper-6 MuiPaper-elevation5 MuiPaper-rounded"
    >
      <div
        className="MuiGrid-root makeStyles-grid-10 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <button
            className="MuiButtonBase-root MuiIconButton-root"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </span>
          </button>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        />
      </div>
    </div>
  </div>
  <div
    className="makeStyles-header-5"
  >
    <div
      className="makeStyles-container-3"
    >
      <div
        className="MuiGrid-root makeStyles-buttonContainer-4 MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              <p
                className="MuiTypography-root makeStyles-buttonFont-2 MuiTypography-body1"
              >
                Stop
              </p>
            </span>
          </button>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              <p
                className="MuiTypography-root makeStyles-buttonFont-2 MuiTypography-body1"
              >
                Clear
              </p>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
